scene/realistic-light-and-shadows/src/main/java/com/esri/samples/realistic_lighting_and_shadows/RealisticLightingAndShadowsSample.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright 2020 Esri.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package com.esri.samples.realistic_lighting_and_shadows;
+
+import java.io.IOException;
+
+import javafx.application.Application;
+import javafx.fxml.FXMLLoader;
+import javafx.scene.Parent;
+import javafx.scene.Scene;
+import javafx.stage.Stage;
+
+public class RealisticLightingAndShadowsSample extends Application {
+
+  private static RealisticLightingAndShadowsController controller;
+
+  @Override
+  public void start(Stage stage) throws IOException {
+
+    // set up the scene
+    FXMLLoader loader = new FXMLLoader(getClass().getResource("/realistic_lighting_and_shadows/main.fxml"));
+    Parent root = loader.load();
+    controller = loader.getController();
+    Scene scene = new Scene(root);
+
+    // set title, size, and add JavaFX scene to stage
+    stage.setTitle("Realistic Environmental Atmosphere Sample");
+    stage.setWidth(800);
+    stage.setHeight(700);
+    stage.setScene(scene);
+    stage.show();
+  }
+
+  /**
+   * /**
+   * Stops and releases all resources used in application.
+   */
```

Comment:
Something slightly odd happened with this JavaDoc:
```suggestion
  /**
   * Stops and releases all resources used in application.
   */
```